catalog/spatial/registry/pom.xml

```diff
@@ -68,6 +68,11 @@
             </dependency>
             <dependency>
                 <groupId>org.codice.ddf.registry</groupId>
+                <artifactId>registry-federation-admin-api</artifactId>
```

Comment:
There are other modules that use the admin api, they just haven't made it over from the other repo yet.